nucypher/cli/stake.py

```diff
@@ -164,7 +180,54 @@ def stake(click_config,
                                          password=password,
                                          worker_address=worker_address)
 
-        emitter.echo(f"OK | Receipt: {receipt['transactionHash'].hex()}", color='green')
+        # TODO: Double-check dates
+        current_period = STAKEHOLDER.staking_agent.get_current_period()
+        bounded_date = datetime_at_period(period=current_period)
```

Comment:
is it meant to be bonded?

Reply:
I think, `bounded`?

Reply:
Ah very confusing: there's `bonded`, `bound`, and `bounded`. But yes, it is meant to be "bonded".